From d021d2778c8bfbc1a08b69a652f4f8818a561cb0 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Tue, 29 Aug 2017 17:49:16 +0300
Subject: [PATCH 406/741] staging: fsl-dpaa2/eth: Add DCB Kconfig

commit cb373a850006f91f87be4aa3deb6a4939f1e6d6d from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add a Kconfig option for enabling Data Center Bridging (DCB)
support in the Ethernet driver. This is needed for priority
flow control support.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index b2064eebd82a..dedb0ed0c135 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -36,6 +36,16 @@ config FSL_DPAA2_ETH_DEBUGFS
 	default n
 	---help---
 	  Enable advanced statistics through debugfs interface.
+
+config FSL_DPAA2_ETH_DCB
+        bool "Data Center Bridging (DCB) Support"
+        default n
+        depends on DCB
+        ---help---
+          Say Y here if you want to use Data Center Bridging (DCB) features
+          (PFC) in the driver.
+
+          If unsure, say N.
 endif
 
 source "drivers/staging/fsl-dpaa2/mac/Kconfig"
-- 
2.17.1

